df.ui.xml

```diff
@@ -566,17 +566,12 @@
 
             <int32_t name='unk6df4'/>
 
-            <stl-vector name='unk_44_12i' since='v0.44.12'/>
-            <stl-vector name='unk_44_12j' since='v0.44.12'/>
-            <stl-vector name='unk_44_12a' since='v0.44.12'/>
+            <bool name='unk_44_12a'/>
+            <padding size='3' comment='workaround for strange IDA 7.0 bug'/>
 
-            <int64_t name='unk_44_12b' since='v0.44.12' init-value='0'/>
-
-            <int32_t name='unk_44_12c' init-value='-1' since='v0.44.12'/>
-            <int32_t name='unk_44_12d' init-value='-1' since='v0.44.12'/>
-
-            <bool name='unk_44_12e' since='v0.44.12'/>
-            <int32_t name='unk_44_12f' since='v0.44.12' comment='Not visible in initial xrefs'/>
+            <compound name='unk_44_12b' type-name='nemesis_offload'/>
+            <bool name='unk_44_12c' since='v0.44.12'/>
+            <int32_t name='unk_44_12d' comment='padding?'/>
```

Comment:
I'm guessing this is equivalent to my int64_t fixing 32-bit alignment. Out of curiosity, did you check that? I can probably do that tomorrow otherwise.

Reply:
Does it mean padding is needed just for IDA, not for compiler?

Reply:
The compiler clearly cares about something here, or this field (or my int64) wouldn't have been needed for 32-bit. I assume you're talking about the `<padding size=3>` above?

Reply:
Well it says workaround for IDA specifically...

Reply:
Right, I'm commenting on another field

Reply:
The variable "unk_44_12d" was previously marked as "Not visible in initial xrefs", which I changed to "padding?" for some reason I can't recall, probably because I couldn't even find any _relative_ references to it (e.g. "mov eax, [rdx+8B9Ch]"), let alone absolute ones.

The problem with IDA was fixed by the "<padding>" inserted immediately before the nemesis_offload compound.